deployment/v3/utils/ssl_renewel_util/certs_renew_cron.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# script is the schedule cron to run the certs renewal script on the expiry date.
+
+# Prompt the user for the expiry date
+read -p "Enter the expiry date (YYYY-MM-DD): " expiry_date
```

Comment:
use the command in the same script to fetch the expiry date for the domain

Reply:
Updated